{"uid":"9be73aae28de6f42","name":"test_dynamic_steps_github","fullName":"tests.test_dynamic_steps#test_dynamic_steps_github","historyId":"e6a8af64436b4742d95cd90079a9cf04","time":{"start":1664809290673,"stop":1664809291411,"duration":738},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x561b57a5f693 <unknown>\n#1 0x561b57858b0a <unknown>\n#2 0x561b5787def2 <unknown>\n#3 0x561b578791f4 <unknown>\n#4 0x561b578b4953 <unknown>\n#5 0x561b578ae743 <unknown>\n#6 0x561b57884533 <unknown>\n#7 0x561b57885715 <unknown>\n#8 0x561b57aaf7bd <unknown>\n#9 0x561b57ab2bf9 <unknown>\n#10 0x561b57a94f2e <unknown>\n#11 0x561b57ab39b3 <unknown>\n#12 0x561b57a88e4f <unknown>\n#13 0x561b57ad2ea8 <unknown>\n#14 0x561b57ad3052 <unknown>\n#15 0x561b57aed71f <unknown>\n#16 0x7f53528a2609 <unknown>","statusTrace":"@allure.tag('web')\n    @allure.severity(Severity.CRITICAL)\n    @allure.label('owner', 'Fatalwgx')\n    @allure.feature('Repository issues accessability')\n    @allure.story(\"First issue's name is \\'This issue is better be found\\'\")\n    @allure.link('https://github.com', name='Testing')\n    def test_dynamic_steps_github():\n        with allure.step('Opening github homepage'):\n>           browser.open(\"https://github.com\")\n\ntests/test_dynamic_steps.py:17: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selene/support/shared/browser.py:63: in open\n    self.config.get_or_create_driver().set_window_size(\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selene/support/shared/config.py:301: in get_or_create_driver\n    return self._source.get_or_create()\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selene/support/shared/config.py:135: in get_or_create\n    return self.create()\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selene/support/shared/config.py:116: in create\n    self._stored = self._set_driver()\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selene/support/shared/config.py:203: in <lambda>\n    lambda: self._set_drivers_from_webdriver_manager()\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selene/support/shared/config.py:277: in _set_drivers_from_webdriver_manager\n    return {\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selene/support/shared/config.py:252: in get_chrome\n    return Chrome(\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selenium/webdriver/chrome/webdriver.py:69: in __init__\n    super().__init__(DesiredCapabilities.CHROME['browserName'], \"goog\",\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selenium/webdriver/chromium/webdriver.py:92: in __init__\n    super().__init__(\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:270: in __init__\n    self.start_session(capabilities, browser_profile)\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:363: in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:428: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f6d5084d840>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: Chrome failed to start: exited a...\\\\n#14 0x561b57ad3052 \\\\u003Cunknown>\\\\n#15 0x561b57aed71f \\\\u003Cunknown>\\\\n#16 0x7f53528a2609 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"\n        Checks that a JSON response from the WebDriver does not have an error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get('status', None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get('value', None)\n            if value_json and isinstance(value_json, str):\n                import json\n                try:\n                    value = json.loads(value_json)\n                    if len(value.keys()) == 1:\n                        value = value['value']\n                    status = value.get('error', None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get('message')\n                    else:\n                        message = value.get('message', None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        if status in ErrorCode.NO_SUCH_ELEMENT:\n            exception_class = NoSuchElementException\n        elif status in ErrorCode.NO_SUCH_FRAME:\n            exception_class = NoSuchFrameException\n        elif status in ErrorCode.NO_SUCH_SHADOW_ROOT:\n            exception_class = NoSuchShadowRootException\n        elif status in ErrorCode.NO_SUCH_WINDOW:\n            exception_class = NoSuchWindowException\n        elif status in ErrorCode.STALE_ELEMENT_REFERENCE:\n            exception_class = StaleElementReferenceException\n        elif status in ErrorCode.ELEMENT_NOT_VISIBLE:\n            exception_class = ElementNotVisibleException\n        elif status in ErrorCode.INVALID_ELEMENT_STATE:\n            exception_class = InvalidElementStateException\n        elif status in ErrorCode.INVALID_SELECTOR \\\n                or status in ErrorCode.INVALID_XPATH_SELECTOR \\\n                or status in ErrorCode.INVALID_XPATH_SELECTOR_RETURN_TYPER:\n            exception_class = InvalidSelectorException\n        elif status in ErrorCode.ELEMENT_IS_NOT_SELECTABLE:\n            exception_class = ElementNotSelectableException\n        elif status in ErrorCode.ELEMENT_NOT_INTERACTABLE:\n            exception_class = ElementNotInteractableException\n        elif status in ErrorCode.INVALID_COOKIE_DOMAIN:\n            exception_class = InvalidCookieDomainException\n        elif status in ErrorCode.UNABLE_TO_SET_COOKIE:\n            exception_class = UnableToSetCookieException\n        elif status in ErrorCode.TIMEOUT:\n            exception_class = TimeoutException\n        elif status in ErrorCode.SCRIPT_TIMEOUT:\n            exception_class = TimeoutException\n        elif status in ErrorCode.UNKNOWN_ERROR:\n            exception_class = WebDriverException\n        elif status in ErrorCode.UNEXPECTED_ALERT_OPEN:\n            exception_class = UnexpectedAlertPresentException\n        elif status in ErrorCode.NO_ALERT_OPEN:\n            exception_class = NoAlertPresentException\n        elif status in ErrorCode.IME_NOT_AVAILABLE:\n            exception_class = ImeNotAvailableException\n        elif status in ErrorCode.IME_ENGINE_ACTIVATION_FAILED:\n            exception_class = ImeActivationFailedException\n        elif status in ErrorCode.MOVE_TARGET_OUT_OF_BOUNDS:\n            exception_class = MoveTargetOutOfBoundsException\n        elif status in ErrorCode.JAVASCRIPT_ERROR:\n            exception_class = JavascriptException\n        elif status in ErrorCode.SESSION_NOT_CREATED:\n            exception_class = SessionNotCreatedException\n        elif status in ErrorCode.INVALID_ARGUMENT:\n            exception_class = InvalidArgumentException\n        elif status in ErrorCode.NO_SUCH_COOKIE:\n            exception_class = NoSuchCookieException\n        elif status in ErrorCode.UNABLE_TO_CAPTURE_SCREEN:\n            exception_class = ScreenshotException\n        elif status in ErrorCode.ELEMENT_CLICK_INTERCEPTED:\n            exception_class = ElementClickInterceptedException\n        elif status in ErrorCode.INSECURE_CERTIFICATE:\n            exception_class = InsecureCertificateException\n        elif status in ErrorCode.INVALID_COORDINATES:\n            exception_class = InvalidCoordinatesException\n        elif status in ErrorCode.INVALID_SESSION_ID:\n            exception_class = InvalidSessionIdException\n        elif status in ErrorCode.UNKNOWN_METHOD:\n            exception_class = UnknownMethodException\n        else:\n            exception_class = WebDriverException\n        if not value:\n            value = response['value']\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and 'message' in value:\n            message = value['message']\n    \n        screen = None  # type: ignore[assignment]\n        if 'screen' in value:\n            screen = value['screen']\n    \n        stacktrace = None\n        st_value = value.get('stackTrace') or value.get('stacktrace')\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split('\\n')\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get('methodName', '<anonymous>')\n                        if 'className' in frame:\n                            meth = \"{}.{}\".format(frame['className'], meth)\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if 'data' in value:\n                alert_text = value['data'].get('text')\n            elif 'alert' in value:\n                alert_text = value['alert'].get('text')\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\nE         (unknown error: DevToolsActivePort file doesn't exist)\nE         (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nE       Stacktrace:\nE       #0 0x561b57a5f693 <unknown>\nE       #1 0x561b57858b0a <unknown>\nE       #2 0x561b5787def2 <unknown>\nE       #3 0x561b578791f4 <unknown>\nE       #4 0x561b578b4953 <unknown>\nE       #5 0x561b578ae743 <unknown>\nE       #6 0x561b57884533 <unknown>\nE       #7 0x561b57885715 <unknown>\nE       #8 0x561b57aaf7bd <unknown>\nE       #9 0x561b57ab2bf9 <unknown>\nE       #10 0x561b57a94f2e <unknown>\nE       #11 0x561b57ab39b3 <unknown>\nE       #12 0x561b57a88e4f <unknown>\nE       #13 0x561b57ad2ea8 <unknown>\nE       #14 0x561b57ad3052 <unknown>\nE       #15 0x561b57aed71f <unknown>\nE       #16 0x7f53528a2609 <unknown>\n\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:243: WebDriverException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"open_browser","time":{"start":1664809290673,"stop":1664809290673,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x561b57a5f693 <unknown>\n#1 0x561b57858b0a <unknown>\n#2 0x561b5787def2 <unknown>\n#3 0x561b578791f4 <unknown>\n#4 0x561b578b4953 <unknown>\n#5 0x561b578ae743 <unknown>\n#6 0x561b57884533 <unknown>\n#7 0x561b57885715 <unknown>\n#8 0x561b57aaf7bd <unknown>\n#9 0x561b57ab2bf9 <unknown>\n#10 0x561b57a94f2e <unknown>\n#11 0x561b57ab39b3 <unknown>\n#12 0x561b57a88e4f <unknown>\n#13 0x561b57ad2ea8 <unknown>\n#14 0x561b57ad3052 <unknown>\n#15 0x561b57aed71f <unknown>\n#16 0x7f53528a2609 <unknown>","statusTrace":"@allure.tag('web')\n    @allure.severity(Severity.CRITICAL)\n    @allure.label('owner', 'Fatalwgx')\n    @allure.feature('Repository issues accessability')\n    @allure.story(\"First issue's name is \\'This issue is better be found\\'\")\n    @allure.link('https://github.com', name='Testing')\n    def test_dynamic_steps_github():\n        with allure.step('Opening github homepage'):\n>           browser.open(\"https://github.com\")\n\ntests/test_dynamic_steps.py:17: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selene/support/shared/browser.py:63: in open\n    self.config.get_or_create_driver().set_window_size(\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selene/support/shared/config.py:301: in get_or_create_driver\n    return self._source.get_or_create()\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selene/support/shared/config.py:135: in get_or_create\n    return self.create()\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selene/support/shared/config.py:116: in create\n    self._stored = self._set_driver()\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selene/support/shared/config.py:203: in <lambda>\n    lambda: self._set_drivers_from_webdriver_manager()\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selene/support/shared/config.py:277: in _set_drivers_from_webdriver_manager\n    return {\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selene/support/shared/config.py:252: in get_chrome\n    return Chrome(\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selenium/webdriver/chrome/webdriver.py:69: in __init__\n    super().__init__(DesiredCapabilities.CHROME['browserName'], \"goog\",\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selenium/webdriver/chromium/webdriver.py:92: in __init__\n    super().__init__(\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:270: in __init__\n    self.start_session(capabilities, browser_profile)\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:363: in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:428: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f6d5084d840>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: Chrome failed to start: exited a...\\\\n#14 0x561b57ad3052 \\\\u003Cunknown>\\\\n#15 0x561b57aed71f \\\\u003Cunknown>\\\\n#16 0x7f53528a2609 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"\n        Checks that a JSON response from the WebDriver does not have an error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get('status', None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get('value', None)\n            if value_json and isinstance(value_json, str):\n                import json\n                try:\n                    value = json.loads(value_json)\n                    if len(value.keys()) == 1:\n                        value = value['value']\n                    status = value.get('error', None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get('message')\n                    else:\n                        message = value.get('message', None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        if status in ErrorCode.NO_SUCH_ELEMENT:\n            exception_class = NoSuchElementException\n        elif status in ErrorCode.NO_SUCH_FRAME:\n            exception_class = NoSuchFrameException\n        elif status in ErrorCode.NO_SUCH_SHADOW_ROOT:\n            exception_class = NoSuchShadowRootException\n        elif status in ErrorCode.NO_SUCH_WINDOW:\n            exception_class = NoSuchWindowException\n        elif status in ErrorCode.STALE_ELEMENT_REFERENCE:\n            exception_class = StaleElementReferenceException\n        elif status in ErrorCode.ELEMENT_NOT_VISIBLE:\n            exception_class = ElementNotVisibleException\n        elif status in ErrorCode.INVALID_ELEMENT_STATE:\n            exception_class = InvalidElementStateException\n        elif status in ErrorCode.INVALID_SELECTOR \\\n                or status in ErrorCode.INVALID_XPATH_SELECTOR \\\n                or status in ErrorCode.INVALID_XPATH_SELECTOR_RETURN_TYPER:\n            exception_class = InvalidSelectorException\n        elif status in ErrorCode.ELEMENT_IS_NOT_SELECTABLE:\n            exception_class = ElementNotSelectableException\n        elif status in ErrorCode.ELEMENT_NOT_INTERACTABLE:\n            exception_class = ElementNotInteractableException\n        elif status in ErrorCode.INVALID_COOKIE_DOMAIN:\n            exception_class = InvalidCookieDomainException\n        elif status in ErrorCode.UNABLE_TO_SET_COOKIE:\n            exception_class = UnableToSetCookieException\n        elif status in ErrorCode.TIMEOUT:\n            exception_class = TimeoutException\n        elif status in ErrorCode.SCRIPT_TIMEOUT:\n            exception_class = TimeoutException\n        elif status in ErrorCode.UNKNOWN_ERROR:\n            exception_class = WebDriverException\n        elif status in ErrorCode.UNEXPECTED_ALERT_OPEN:\n            exception_class = UnexpectedAlertPresentException\n        elif status in ErrorCode.NO_ALERT_OPEN:\n            exception_class = NoAlertPresentException\n        elif status in ErrorCode.IME_NOT_AVAILABLE:\n            exception_class = ImeNotAvailableException\n        elif status in ErrorCode.IME_ENGINE_ACTIVATION_FAILED:\n            exception_class = ImeActivationFailedException\n        elif status in ErrorCode.MOVE_TARGET_OUT_OF_BOUNDS:\n            exception_class = MoveTargetOutOfBoundsException\n        elif status in ErrorCode.JAVASCRIPT_ERROR:\n            exception_class = JavascriptException\n        elif status in ErrorCode.SESSION_NOT_CREATED:\n            exception_class = SessionNotCreatedException\n        elif status in ErrorCode.INVALID_ARGUMENT:\n            exception_class = InvalidArgumentException\n        elif status in ErrorCode.NO_SUCH_COOKIE:\n            exception_class = NoSuchCookieException\n        elif status in ErrorCode.UNABLE_TO_CAPTURE_SCREEN:\n            exception_class = ScreenshotException\n        elif status in ErrorCode.ELEMENT_CLICK_INTERCEPTED:\n            exception_class = ElementClickInterceptedException\n        elif status in ErrorCode.INSECURE_CERTIFICATE:\n            exception_class = InsecureCertificateException\n        elif status in ErrorCode.INVALID_COORDINATES:\n            exception_class = InvalidCoordinatesException\n        elif status in ErrorCode.INVALID_SESSION_ID:\n            exception_class = InvalidSessionIdException\n        elif status in ErrorCode.UNKNOWN_METHOD:\n            exception_class = UnknownMethodException\n        else:\n            exception_class = WebDriverException\n        if not value:\n            value = response['value']\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and 'message' in value:\n            message = value['message']\n    \n        screen = None  # type: ignore[assignment]\n        if 'screen' in value:\n            screen = value['screen']\n    \n        stacktrace = None\n        st_value = value.get('stackTrace') or value.get('stacktrace')\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split('\\n')\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get('methodName', '<anonymous>')\n                        if 'className' in frame:\n                            meth = \"{}.{}\".format(frame['className'], meth)\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if 'data' in value:\n                alert_text = value['data'].get('text')\n            elif 'alert' in value:\n                alert_text = value['alert'].get('text')\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\nE         (unknown error: DevToolsActivePort file doesn't exist)\nE         (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nE       Stacktrace:\nE       #0 0x561b57a5f693 <unknown>\nE       #1 0x561b57858b0a <unknown>\nE       #2 0x561b5787def2 <unknown>\nE       #3 0x561b578791f4 <unknown>\nE       #4 0x561b578b4953 <unknown>\nE       #5 0x561b578ae743 <unknown>\nE       #6 0x561b57884533 <unknown>\nE       #7 0x561b57885715 <unknown>\nE       #8 0x561b57aaf7bd <unknown>\nE       #9 0x561b57ab2bf9 <unknown>\nE       #10 0x561b57a94f2e <unknown>\nE       #11 0x561b57ab39b3 <unknown>\nE       #12 0x561b57a88e4f <unknown>\nE       #13 0x561b57ad2ea8 <unknown>\nE       #14 0x561b57ad3052 <unknown>\nE       #15 0x561b57aed71f <unknown>\nE       #16 0x7f53528a2609 <unknown>\n\n/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:243: WebDriverException","steps":[{"name":"Opening github homepage","time":{"start":1664809290674,"stop":1664809291410,"duration":736},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x561b57a5f693 <unknown>\n#1 0x561b57858b0a <unknown>\n#2 0x561b5787def2 <unknown>\n#3 0x561b578791f4 <unknown>\n#4 0x561b578b4953 <unknown>\n#5 0x561b578ae743 <unknown>\n#6 0x561b57884533 <unknown>\n#7 0x561b57885715 <unknown>\n#8 0x561b57aaf7bd <unknown>\n#9 0x561b57ab2bf9 <unknown>\n#10 0x561b57a94f2e <unknown>\n#11 0x561b57ab39b3 <unknown>\n#12 0x561b57a88e4f <unknown>\n#13 0x561b57ad2ea8 <unknown>\n#14 0x561b57ad3052 <unknown>\n#15 0x561b57aed71f <unknown>\n#16 0x7f53528a2609 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Allure_representation_github/Allure_representation_github/tests/test_dynamic_steps.py\", line 17, in test_dynamic_steps_github\n    browser.open(\"https://github.com\")\n  File \"/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selene/support/shared/browser.py\", line 63, in open\n    self.config.get_or_create_driver().set_window_size(\n  File \"/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selene/support/shared/config.py\", line 301, in get_or_create_driver\n    return self._source.get_or_create()\n  File \"/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selene/support/shared/config.py\", line 135, in get_or_create\n    return self.create()\n  File \"/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selene/support/shared/config.py\", line 116, in create\n    self._stored = self._set_driver()\n  File \"/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selene/support/shared/config.py\", line 203, in <lambda>\n    lambda: self._set_drivers_from_webdriver_manager()\n  File \"/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selene/support/shared/config.py\", line 277, in _set_drivers_from_webdriver_manager\n    return {\n  File \"/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selene/support/shared/config.py\", line 252, in get_chrome\n    return Chrome(\n  File \"/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selenium/webdriver/chrome/webdriver.py\", line 69, in __init__\n    super().__init__(DesiredCapabilities.CHROME['browserName'], \"goog\",\n  File \"/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selenium/webdriver/chromium/webdriver.py\", line 92, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 270, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 363, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n  File \"/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 428, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.10.7/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 243, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"attachments":[],"parameters":[],"stepsCount":1,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[],"labels":[{"name":"owner","value":"Fatalwgx"},{"name":"story","value":"First issue's name is 'This issue is better be found'"},{"name":"feature","value":"Repository issues accessability"},{"name":"severity","value":"critical"},{"name":"tag","value":"web"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_dynamic_steps"},{"name":"host","value":"fv-az220-425"},{"name":"thread","value":"2296-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_dynamic_steps"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[{"name":"Testing","url":"https://github.com","type":"link"}],"hidden":false,"retry":false,"extra":{"severity":"critical","owner":"Fatalwgx","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":2,"skipped":0,"passed":0,"unknown":0,"total":2},"items":[{"uid":"422ba00a0269af1a","reportUrl":"https://Fatalwgx.github.io/Allure_representation_github/1//#testresult/422ba00a0269af1a","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x565201051693 <unknown>\n#1 0x565200e4ab0a <unknown>\n#2 0x565200e6fef2 <unknown>\n#3 0x565200e6b1f4 <unknown>\n#4 0x565200ea6953 <unknown>\n#5 0x565200ea0743 <unknown>\n#6 0x565200e76533 <unknown>\n#7 0x565200e77715 <unknown>\n#8 0x5652010a17bd <unknown>\n#9 0x5652010a4bf9 <unknown>\n#10 0x565201086f2e <unknown>\n#11 0x5652010a59b3 <unknown>\n#12 0x56520107ae4f <unknown>\n#13 0x5652010c4ea8 <unknown>\n#14 0x5652010c5052 <unknown>\n#15 0x5652010df71f <unknown>\n#16 0x7f92fdd63609 <unknown>","time":{"start":1664808837211,"stop":1664808837965,"duration":754}}]},"tags":["web"]},"source":"9be73aae28de6f42.json","parameterValues":[]}